queries/models/transito/schema.yml

```
@@ -59,8 +59,162 @@ models:
+
+  - name: aux_autuacao_id
+    description: "Tabela auxiliar para gerar identificador único da autuação"
+    columns:
+      - name: data
+        description: "Data da autuação (Coluna de particionamento)"
+      - name: id_autuacao
+        description: "Identificador único da autuação"
+      - name: id_auto_infracao
+        description: "Código do auto de infração [protegido]"
+        policy_tags:
+        - 'projects/rj-smtr/locations/us/taxonomies/7968932463054912793/policyTags/5243840798726507169'
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 10 but found 8

(indentation)